content/en/_index.html

```diff
@@ -3,76 +3,47 @@
 linkTitle: "Welcome to Kuadrant!"
 ---
 
-{{< blocks/cover title="Welcome to the Kuadrant Project!" image_anchor="top" height="full" color="orange" >}}
+{{< blocks/cover title="Securely expose & consume k8s services with Kuadrant" image_anchor="top" height="full" >}}
 <div class="mx-auto">
 	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
 		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
 	</a>
-	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/Kuadrant">
-		Download <i class="fab fa-github ml-2 "></i>
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://operatorhub.io/operator/kuadrant-operator">
+		Install <i class="fas fa-bolt ml-2 "></i>
 	</a>
-	<p class="lead mt-5">Kubernetes Native stuff that you want to use!</p>
-	{{< blocks/link-down color="primary" >}}
+	<p class="lead mt-10">Kuadrant bundles <a href="https://github.com/Kuadrant/limitador">Limitador</a>, the rate limiting service, and <a href="https://github.com/Kuadrant/authorino">Authorino</a>, our lightweight authorization server. The Kuadrant Operator, available from <a href="https://operatorhub.io/operator/kuadrant-operator">Operator Hub</a>, will take care of all the wiring and deployments of the services required for applying Rate Limits and Authorization Policies to your infrastructure.</p><p>Once installed, Policy Custom Resource Definitions let cluster operators apply a cluster-wide Rate Limit policy to protect the infrastructure from DDoS attacks; while application developers can use advanced features to control usages of their API endpoints based on API keys handed out to their users. Kuadrant supports a wide range of use cases, from simple Rate Limiting to ones that span across authorization and identity-based Rate Limits. Kuadrant leverages the <a href="https://gateway-api.sigs.k8s.io/">Kubernetes Gateway API</a></p>
```

Comment:
> application developers can use advanced features to control usages of their API endpoints based on API keys handed out to their users.

I think this is way too specific for the description. Readers may inadvertently think Kuadrant is only API key authentication.

Reply:
```suggestion
	<p class="lead mt-10">Kuadrant bundles <a href="https://github.com/Kuadrant/limitador">Limitador</a>, the rate limiting service, and <a href="https://github.com/Kuadrant/authorino">Authorino</a>, our lightweight authorization agent. The Kuadrant Operator, available from <a href="https://operatorhub.io/operator/kuadrant-operator">Operator Hub</a>, will take care of all the wiring and deployments of the services required for applying Rate Limits and Authorization Policies to your infrastructure.</p><p>Once installed, Policy Custom Resource Definitions let cluster operators apply a cluster-wide Rate Limit policy to protect the infrastructure from DDoS attacks; while application developers can use advanced features to control usages of their API endpoints with tailor-made authentication and authorization rules and protocols. Kuadrant supports a wide range of use cases, from simple Rate Limiting to ones that span across authorization and identity-based Rate Limits. Kuadrant leverages the <a href="https://gateway-api.sigs.k8s.io/">Kubernetes Gateway API</a></p>
```